timemachine/fe/free_energy.py

```diff
@@ -455,7 +439,39 @@ def run_production_local_steps(n_steps: int) -> Tuple[NDArray, NDArray, NDArray]
 
     assert np.all(np.isfinite(all_coords[-1])), "Production resulted in a nan"
 
-    return Trajectory(all_coords, all_boxes, final_velocities)
+    final_barostat_volume_scale_factor = ctxt.get_barostat().get_volume_scale_factor() if ctxt.get_barostat() else None
+
+    return Trajectory(all_coords, all_boxes, final_velocities, final_barostat_volume_scale_factor)
+
+
+def sample(initial_state: InitialState, md_params: MDParams, max_buffer_frames: int) -> Trajectory:
+    """Generate a trajectory given an initial state and a simulation protocol
+
+    Parameters
+    ----------
+    initial_state: InitialState
+        (contains potentials, integrator, optional barostat)
+
+    md_params: MDParams
+        (specifies x0, v0, box0, number of MD steps, thinning interval, etc...)
```

Comment:
nit[docs] -- I know this wasn't added in this PR, but the descriptions are out of date and can be removed ((x0, v0, box0) are in initial_state, etc., type annotation seems sufficient)

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/1151/commits/0a10e54faa4838208ef51a6ea6fa1894d5c73645